reverse_engineering/databaseService/databaseService.js

```diff
@@ -74,7 +74,8 @@ const getConnectionClient = async (connectionInfo, logger) => {
 			domain: connectionInfo.userDomain,
 			options: {
 				encrypt: connectionInfo.encryptWindowsConnection === undefined ? false : Boolean(connectionInfo.encryptWindowsConnection),
-				enableArithAbort: true
+				enableArithAbort: true,
+          		trustServerCertificate: getSslConfig(connectionInfo).trustServerCertificate
```

Comment:
Could you please move the obtaining of sslConfig to the start of this function because it seems it is used in every connection method? Also, there is incorrect indent on this line